x-pack/plugins/fleet/cypress/integration/integrations.spec.ts

```diff
@@ -30,12 +30,36 @@ describe('Add Integration', () => {
   after(() => {
     deleteIntegrations(integration);
   });
-
   it('should display Apache integration in the Policies list once installed ', () => {
+    addAndVerifyIntegration();
+  });
+
+  function addAndVerifyIntegration() {
     navigateTo(INTEGRATIONS);
     cy.get(INTEGRATIONS_CARD).contains(integration).click();
     addIntegration();
     cy.get(INTEGRATION_NAME_LINK).contains('apache-');
+  }
+
+  it('[Mocked requests] should display Apache integration in the Policies list once installed ', () => {
```

Comment:
this test is the duplication of the one above, but with mocked requests to showcase network stubbing. 
Pros: quicker test, isolation of UI testing
Cons: lot of extra setup needed (fixture files, intercept each api call)
I think for critical testcases real e2e test would be better, mocked requests are useful when the goal is to test complex UI logic without relying on real backend.
See https://docs.cypress.io/guides/guides/network-requests